SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSubviews.cpp

```diff
@@ -0,0 +1,220 @@
+#include "Interop/ActorSubviews.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Interop/InitialOnlyFilter.h"
+#include "Schema/Restricted.h"
+#include "Schema/Tombstone.h"
+#include "Schema/UnrealMetadata.h"
+#include "SpatialView/EntityComponentTypes.h"
+#include "SpatialView/EntityView.h"
+#include "SpatialView/SubView.h"
+#include "SpatialView/ViewCoordinator.h"
+
+namespace SpatialGDK
+{
+namespace ActorSubviews
+{
+namespace MainActorSubviewSetup
```

Comment:
I presume one of these namespaces is to avoid an anon one, but why so many others and why prototype each before defining?

Reply:
These are for grouping purposes - I had them in structs before, but @improbable-fabian's suggestion to trade them for namespaces seemed logical.